packages/integration/lib/src/views/activity/activity_screen.dart

```diff
@@ -467,6 +472,217 @@ class _LMFeedActivityScreenState extends State<LMFeedActivityScreen> {
     );
   }
 
+  Widget _defPollWidget(LMFeedPoll pollWidget, LMPostViewData postViewData) {
```

Comment:
Return type should be `LMFeedPollWidget`

Reply:
here return type will be `Widget` since it is origially called in `LMFeedPostMedia` where the return type is `LMFeedPollWidget` on feed_screen and other screens we are just ovverriding them with it's implementation